### DISCUSSION TITLE
New poll alternative view

### DISCUSSION THREAD

**Discussion by elioqoshi** [lukevella/rallly#1186] [Ideas]
Just tried the latest self-hosted version and see some great quality-of-life improvements!

While this lacks research and is more heuristics-based, the new poll view of past polls flows in a zig-zag pattern, which is not ideal for scanning quickly in a linear timeflow. Having an alternate 1-column view like before would be a good idea to offer another more traditional view.

<img width="1185" alt="grafik" src="https://github.com/lukevella/rallly/assets/5436686/1727ed07-69ac-4a2f-bc4f-fecdaef1fdcf">

I'd be curious to understand the context behind this change and user research backing this up (which I lack as of now)

**Comment by lukevella** [comment]
Hey @elioqoshi, I do plan on supporting the list view again and possibly other views as well in the future. The idea behind this design originated from having multiple types of scheduling forms (eg. 1:1, booking pages etc…) that would all appear in the same page. I think it's hard to see the full benefit of it right now since it's incomplete.

I was hoping that presenting the polls in this way would start encouraging users to close inactive polls. It's really hard to gauge the number of truly active polls the average user has, since many are still left open even after a date is decided. If you have any feedback on this, do let me know.

P.S. -  moving to discussions since there aren't any actionable tasks as of yet.
